### ISSUE TITLE
This no longer works

### ISSUE THREAD

**Issue by morhsn** [starsprung/amazon-ynab-sync#35]
Since Amazon has canceled the option to create Order History Reports (https://www.amazon.com/b2b/reports), this no longer works. Such a bummer! This no longer works as of March 2023.

Thank you for @starsprung for making this happen. Hopefully there'll be a way to achieve this again in the future.

**Comment by dlardo** [comment]
Yes this is broken for me as well. @starsprung are you looking for contributors or are you over it?

**Comment by starsprung** [comment]
@dlardo I no longer use YNAB since they increased their pricing, so I'm no longer maintaining this project.

**Comment by davelab6** [comment]
@starsprung please go to https://github.com/starsprung/amazon-ynab-sync/settings and "archive" the project :)
